uberfire-security/uberfire-security-management/uberfire-security-management-api/src/main/java/org/uberfire/ext/security/management/api/exception/EntityAlreadyExistsException.java

```diff
@@ -0,0 +1,35 @@
+package org.uberfire.ext.security.management.api.exception;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+
+/**
+ * <p>Exception for user system management when the entity to create already exists.</p>
+ *
+ * @since 0.8.0
+ */
+@Portable
```

Comment:
Since RuntimeException is serializable, we're getting warnings in all these sub types. Best to add @SuppressWarnings("serial"). We don't care about java.io serializability as we're using errai marshalling anyway.